### PR TITLE
Add ERC-7579 + MSA docs and tg chat

### PR DESCRIPTION
**Pull request by kopy-kat**

### CODE REVIEW

**Comment by randomishwalk** [comment]
hey @kopy-kat do you think we should add https://github.com/ethereum/ERCs/pull/163/files to the "EIP" section? Maybe we can update that section to something like "EIP/ERC"?

What do you think?

**Comment by kopy-kat** [comment]
I was thinking of waiting until the first draft is merged and link to the eip site, but I could also add the link to the pr now and change it once it’s merged

**Comment by randomishwalk** [comment]
got it. ok let's do that, we can just update that link to link to eip/erc website once the first draft is out 👍

**Review by randomishwalk — APPROVED** [review]

**Pull request merged**
